src/rivian/schemas/gateway.graphql

```diff
@@ -46,6 +46,7 @@ type Mutation {
   ): MobileLoginResponse
   logout: LogoutResponse
   resendOTPCode(otpToken: String!): ResendOTPCodeResponse
+  sendTripToVehicle(attrs: SendTripToVehicleInput!) { sendTripToVehicle(attrs: $attrs) }
```

Comment:
This should include the return type, not the query

Reply:
Once this is resolved, it looks good to me to ship

Reply:
what a silly error on my part